src/main/java/ro/utcn/sd/boti/stackoverflow/entity/Answer.java

```diff
@@ -0,0 +1,42 @@
+package ro.utcn.sd.boti.stackoverflow.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Answer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "author")
+    private User author;
+
+    private String text;
+    private String time;
```

Comment:
You should map this column to one of the following: LocalDateTime or java.util.Date or java.sql.Timestamp. I would strongly recommend that you use LocalDateTime, as it is newer (although you might have to do some small extra work to get it running for the JDBC repo(s)).